### PR TITLE
New Storage Driver: Cassandra

### PR DESCRIPTION
**Pull request by aliziyacevik**
Hey guys,this pr aims to add new Storage implementation of Cassandra.

Thanks for the reviews.

### CODE REVIEW

File: cassandra/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Cassandra 
+
+A Cassandra storage driver using [cassandra/mongo-go-driver]("https://github.com/gocql/gocql").
```
Comment (gaby) [inline cassandra/README.md:3]: Should it say mongo-go-driver here?

File: cassandra/go.mod [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module github.com/gofiber/storage/cassandra
+
+go 1.18
```
Comment (gaby) [inline cassandra/go.mod:3]: This should be go 1.19

**Review by gaby — CHANGES_REQUESTED** [review]
Test/release/dependabot files are missing.

File: cassandra/go.mod (hunk 2 above)
Comment (aliziyacevik) [reply cassandra/go.mod:3]: sure. do you need this to be at least 1.19 or exact 1.19 ? @gaby

File: cassandra/README.md (hunk 1 above)
Comment (aliziyacevik) [reply cassandra/README.md]: added all the missing files.

File: .github/release-drafter-cassandra.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+name-template: 'MongoDB - v$RESOLVED_VERSION'
+tag-template: 'mongodb/v$RESOLVED_VERSION'
+tag-prefix: mongodb/v
```
Comment (gaby) [inline .github/release-drafter-cassandra.yml:3]: Wrong name

File: .github/release-drafter-cassandra.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+name-template: 'MongoDB - v$RESOLVED_VERSION'
```
Comment (gaby) [inline .github/release-drafter-cassandra.yml:1]: Same here

File: .github/release-drafter-cassandra.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+tag-template: 'mongodb/v$RESOLVED_VERSION'
+tag-prefix: mongodb/v
+include-paths:
+  - mongodb
```
Comment (gaby) [inline .github/release-drafter-cassandra.yml:5]: And here

File: .github/release-drafter-cassandra.yml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            - 'patch'
+            - '📒 Documentation'
+            - '☢️ Bug'
+            - '🤖 Dependencies'
+            - '🧹 Updates'
+    default: patch
+template: |
+    $CHANGES
+
+    **Full Changelog**: https://github.com/$OWNER/$REPOSITORY/compare/$PREVIOUS_TAG...mongodb/v$RESOLVED_VERSION
```
Comment (gaby) [inline .github/release-drafter-cassandra.yml:41]: And here

File: .github/release-drafter-mongodb.yml [hunk 7]
```diff
@@ -1,8 +1,8 @@
-name-template: 'MongoDB - v$RESOLVED_VERSION'
```
Comment (gaby) [inline .github/release-drafter-mongodb.yml:1]: You are editing the drafter-mongodb.yml, all these changes here need to be revert

File: .github/workflows/test-cassandra.yml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        runs-on: ubuntu-latest
+        services:
+            mongo:
+                image: 'cassandra:latest'
+                ports:
+                    - '27017:27017'
+        strategy:
+            matrix:
+                go-version:
+                    - 1.17.x
```
Comment (gaby) [inline .github/workflows/test-cassandra.yml:23]: Only 1.19.x and 1.20.x are supported by the go library used to talk to Cassandra

File: cassandra/README.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
+# Cassandra 
+
+A Cassandra storage driver using [cassandra/cassandra-go-driver]("https://github.com/gocql/gocql").
```
Comment (gaby) [inline cassandra/README.md:3]: It should say `using [gocql/gocql]`

File: cassandra/go.mod [hunk 10]
```diff
@@ -0,0 +1,12 @@
+module github.com/gofiber/cassandra
+
+go 1.20
```
Comment (gaby) [inline cassandra/go.mod:3]: Go 1.19

**Review by gaby — CHANGES_REQUESTED** [review]

File: .github/release-drafter-cassandra.yml (hunk 3 above)
Comment (aliziyacevik) [reply .github/release-drafter-cassandra.yml:3]: sorry, mixed mongo and Cassandra files.

**Comment by gaby** [comment]
@aliziyacevik Can you fix the merge conflict, and check the tests

File: .github/workflows/test-cassandra.yml [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+            - 'cassandra/**'
+name: "Tests Cassandra"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        services:
+            mongo:
+                image: 'cassandra:latest'
+                ports:
+                    - '27017:27017'
```
Comment (gaby) [inline .github/workflows/test-cassandra.yml:19]: This port is the mongodb one, which could be causing the failure in the Test Suite

**Review by gaby — CHANGES_REQUESTED** [review]

**Comment by gaby** [comment]
@aliziyacevik What happened?

**Pull request closed without merging**